### DISCUSSION TITLE
Multiple choice block (Accept X out of total Y)

### DISCUSSION THREAD

**Discussion by ahmad-punch** [quillforms/quillforms#109] [Q&A]
Is there a way to accept say only 2 answers out of the available multiple choices in the multiple-choice block? How can we make it so we accept only X answers out of Y say top 3 out of 10 etc.

{
  name: "multiple-choice",
  attributes: {
    "choices": { 
      "value":  string;
      "label": string;
    }[];  // Default:  [ { "value": "123e45z7o89b",	"label": "Choice 1" }]
    "verticalAlign": boolean; // Default : false
    "multiple": boolean; // Default : false
  }
}

**Comment by mdmag** [comment]
@ahmad-punch This feature will be added in a next version
